decryptor/base/decryptionNotification.go

```diff
@@ -95,6 +99,7 @@ func ColumnInfoFromContext(ctx context.Context) (ColumnInfo, bool) {
 // DecryptionSubscriber interface to subscribe on column's data in db responses
 type DecryptionSubscriber interface {
 	OnColumn(context.Context, []byte) (context.Context, []byte, error)
+	ID() string
```

Comment:
for what it used? ID of what it should return?

Reply:
> for what it used? ID of what it should return?

ID of the subscriber, it's only used for error reporting right now.